lib/common/logging.c

```diff
@@ -129,9 +130,12 @@ crm_trigger_blackbox(int nsig)
 void
 crm_log_deinit(void)
 {
-    if (glib_log_default != NULL) {
-        g_log_set_default_handler(glib_log_default, NULL);
+    if (pcmk__log_id == 0) {
+        return;
     }
+
+    g_log_remove_handler(G_LOG_DOMAIN, pcmk__log_id);
+    g_log_remove_handler("GLib", pcmk__glib_log_id);
```

Comment:
I don't believe it can - looking at the definition of `g_log_set_handler_full` (https://gitlab.gnome.org/GNOME/glib/-/blob/main/glib/gmessages.c?ref_type=heads#L732) it returns 0 for the handler ID on error and it pre-increments the global `handler_id` variable that it hands out so it should always be starting with 1.

The fact that it returns 0 on error makes me think it can't possibly be used internally to identify the default handler.  Plus, the default handler is referenced by `default_log_func` throughout and not by ID as far as I can tell.